README.adoc

```diff
@@ -163,6 +163,9 @@ include::{common-includes}/trywhatyoubuild-end.adoc[]
 
 Navigate to the `start` directory.
 
+[role=command]
+include::{common-includes}/devmode-start.adoc[]
+
 Begin by building your response JSON, which is composed of the `_links` array, as well as the name of
```

Comment:
I don't think this changed, but I'd suggest updating 
"Begin by building your response JSON, which is composed of the `_links` array, as well as the name of the host machine."
To
"Begin by building your response JSON, which is composed of the `_links` array and the name of
the host machine."